index.html

```diff
@@ -600,6 +600,37 @@ <h4>linkedResource</h4>
     }</pre>
       
       </section>
+
+        <section>
+          <h4>dnsDomain</h4>
+          <table class="simple" style="width: 100%;">
+            <thead>
+              <tr>
+                <th>Normative Definition</th>
+                <th>JSON-LD</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <a
+                    href="https://www.ietf.org/archive/id/draft-carter-high-assurance-dids-with-dns-04.html#name-the-dnsdomain-property">High Assurance DIDs with DNS</a>
+                </td>
+                <td>
+                  <a href="https://ciralabs.github.io/DnsDomainJsonLd/dns-domain.jsonld">dnsDomain JSON-LD
```

Comment:
Hey @jessecarter111 -- this is great, we should approve the registration. Before we do that, however, I'm going to try to convince you to future proof your URLs a bit more. Namely, you are mapping "dnsDomain" to a specific IETF I-D that is guaranteed to change over the next couple of months/years: "https://www.ietf.org/archive/id/draft-carter-high-assurance-dids-with-dns-04.html#section-3.2.1". You are also publishing your JSON-LD context at a specific Github URL (that is also bound to change once this thing goes standards track).

Ideally, your context file location AND your term URL will be stable as things evolve for your spec. We tend to use this site to ensure that happens:

https://w3id.org/

and context URLs like this one:

https://w3id.org/age/v1

and term URLs like this one:

https://w3id.org/age#overAge

... which then are re-directed to the current hosting location. The benefit of going that route is that your context URLs and term URLs are stable. I'll also note that we've been burned by publishing a "v1" URL too early -- people other than us ship it into production and then we can't change it because other people shipped it into production. So, I suggest something that no one is going to understand as "in production"... unless you really feel like you're done.

To summarize, I suggest something like:

https://w3id.org/hadids/v0.3

with vocabulary URLs like:

https://w3id.org/hadids#dnsDomain

Finally, the `dnsDomain` term seems adequate, but I'm wondering if you've thought about what you want the relationship to really express. It reads like this:

<DID> dnsDomain "mydomain.example" .

So, a straightfoward reading is: "This DID's DNS domain is mydomain.example." -- which is fine, but it really doesn't convey that there is way more to it than that. What you're really trying to convey is: "There is a High Assurance protocol that you can run against this domain that will tell you whether or not this DID document is secured." Some (inadequate) alternatives might include:

* `highAssuranceDomain`
* `dnssecVerificationDomain`
* `dnssecValidation`

... or something else that would say something more than just "this domain is associated with this DID." Now, to be fair, if the term is `https://w3id.org/hadids#dnsDomain` and that's strongly bound to the HA DID DNS protocol, then that's probably enough. Just trying to think through the developer ergonomics here.

All that said... food for thought, let us know if you want to take action on that, and if not, we can let the registration through. I do hope, however, that you consider the alternatives as they'll help you not break early adopter implementations as this (very exciting) feature evolves.

Reply:
I will definitely be taking action on those suggestions and have the updates through soon. Thank you @msporny, the input is appreciated!

Reply:
Pushed updates to the linked URLs, which now use the W3ID base. Also changed the property name to "dnsValidationDomain" to better reflect the intent and functionality of what accompanies the relationship expressed by the property.